scripts/coverage/run_all_coverage.sh

```diff
@@ -0,0 +1,92 @@
+#!/bin/sh
+#
+# Copyright (c) 2020 Seagate Technology LLC and/or its Affiliates
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# For any questions about this software or licensing,
+# please email opensource@seagate.com or cortx-questions@seagate.com.
+#
+
+set -e
+set +o posix
+
+# variables
+SCRIPT_PATH=$(readlink -f "$0")
+SCRIPT_DIR=$(dirname "$SCRIPT_PATH")
+SRC_DIR="$(dirname "$(dirname "$SCRIPT_DIR" )")"
+#default codacy project token
+CODACY_PROJECT_TOKEN=7a64ef82e1a049ab898fe838f1320594
+BRANCH="main"
+
+export DES_DIR="/root/code_coverage"
```

Comment:
The default should be within the project itself - let's not do it in the root

Reply:
Try - /root/workspace/cortx-s3server/.coverage 
Also - take this as a parameter if someone wants overwrite

Reply:
added and changed to project directory.
Done